### DISCUSSION TITLE
Stripe Payout is stuck for 3 months now

### DISCUSSION THREAD

**Discussion by kraanzu** [community/community#125030] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I haven't received my sponsorship amount for more than 3 months now. It always says that the payout will happen on <month_name> 22nd but then after the 22nd, It'll postpone the date to the next month

![image](https://github.com/community/community/assets/97718086/179dfcbb-b368-4fb2-8e0b-4d92b90c976f)

**Comment by kraanzu** [comment]
One more thing to add: It's not a new account, I was receiving payouts until feb 22nd

**Comment by kraanzu** [comment]
Hey @antn and @slincoln75,

I saw on other discussions that you can also do manual payment in case the sponsor gets stuck. Would that be possible in my case?
I have also opened a tick on stripe support so I'll also update in case I get a reply

Thanks!

**Reply by ebndev** [reply]
Hi @kraanzu, thanks for being a part of the GitHub Community!

For answers to account-specific questions like this one, please open a ticket on our [Support page](https://support.github.com/contact). The support team will be able to look into the reason your payment is stuck and process a manual payment if that is the appropriate solution. 

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

Thanks for understanding!

**Reply by kraanzu** [reply]
Hi @ebndev, I already opened the ticket on support but it has been over a month and I haven't received a response :(

Ticket number: 2743771

As I mentioned, It's been over 3 months so I hope this can be resolved soon.
Thanks

**Reply by kraanzu** [reply]
I also opened a ticket regarding GitHub education before and didn't get a response back for a month as well, hence the reason for opening a discussion here

**Reply by ShafSpecs** [reply]
> Hi @kraanzu, thanks for being a part of the GitHub Community!
> 
> For answers to account-specific questions like this one, please open a ticket on our [Support page](https://support.github.com/contact). The support team will be able to look into the reason your payment is stuck and process a manual payment if that is the appropriate solution.
> 
> This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.
> 
> Thanks for understanding!

Hello @ebndev, same issue and same process here as well. I have the very same issue with my sponsors as kraanzu. I opened a ticket (ID: #2808380 if that helps) and have not received a response at all from Github. I noticed a very few were able to get their payout manually triggered. Was wondering if it's possible to have something like that for the sponsored accounts themselves? Being able to open a stripe ticket to trigger a manual payout if github payout hangs for over a month. It seems to be a common (and very frustrating especially when it's your main income) issue here, and I know due to Github size and nature, resolving tickets might take a year or more.

**Reply by catsintents** [reply, marked as answer]
Hi folks! Engineer on the Sponsors team. Thanks for opening these tickets. I checked and they are still in the queue so rest assured they will be worked on, Thanks!

**Reply by kraanzu** [reply]
Thanks for the update @catsintents !

What are your thoughts on @ShafSpecs's idea?  We also noticed that this is a common issue with a lot of people and we can't open support tickets every time if it happens to us and wait for them to be solved

Definitely not waiting this long:
> and I know due to Github size and nature, resolving tickets might take a year or more.

**Reply by kraanzu** [reply]
I was able to get my payment manually! Thank you :)
What about you @ShafSpecs ?

**Reply by ShafSpecs** [reply]
> I was able to get my payment manually! Thank you :) What about you @ShafSpecs ?

It hasn't been remitted yet, but my stripe dashboard shows the transfer to my bank is pending. Thanks to all who made it happen :)

**Reply by kraanzu** [reply]
Yes same! Glad it worked for you as well!

**Reply by kraanzu** [reply]
I'll close this issue for now then

**Reply by catsintents** [reply]
Glad folks are getting these approvals! We had someone go through and push these through. 🚀

**Reply by kraanzu** [reply]
What caused the issue exactly? can it happen again? can we prevent it if that's the case??

**Reply by theonlymo** [reply]
Same for me. My payouts usually get into my bank on the 22th, but nothing is triggered this month. (ticket  #2896252)
I also noticed that the email wording from sponsorships is different. in the past it used to look like this 
![image](https://github.com/user-attachments/assets/b9771c59-36ed-48f7-b5a5-6a4b7ba56216)

now it's like this 
![image](https://github.com/user-attachments/assets/85d11a77-370b-43a2-9327-3cfbb905b02a)

Do you have any idea what changed?

**Reply by cuilei5205189** [reply]
> Hi folks! Engineer on the Sponsors team. Thanks for opening these tickets. I checked and they are still in the queue so rest assured they will be worked on, Thanks!

Hi @catsintents,

I’m experiencing a similar GitHub Sponsors payout issue.

My Sponsors dashboard estimated payout has rolled over to the next month, and in Stripe Express the balance is already available, but there is no payout pending or in transit.

I have already opened a GitHub Support ticket: #4508632, but I haven’t received a response yet.

Could you please help check this or route it to the GitHub Sponsors team? It looks like the payout was not initiated rather than delayed by the bank.

Thank you.

**Comment by Rana-Saud** [comment]
help us

**Comment by starriv** [comment]
Hi, I have the same situation @catsintents 

I opened a support ticket, but no one responded to me for 3 weeks.

**Reply by kraanzu** [reply]
They are really taking a long time, wonder whats causing the issue
I reached out to stripe team about it and they said that they cant do anything related to this

**Reply by ebndev** [reply]
Hi @starriv, your ticket is in the support queue and is in review by the Sponsors team. They will update here or in your ticket once reviewed!

**Reply by ebndev** [reply]
@kraanzu @starriv 

In the future, please note: contacting any member of GitHub Staff via unsolicited mentions or pings is strongly discouraged and may be considered a violation of our prohibition against harassment. Please review our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) for more information.

Thanks for understanding!

**Reply by kraanzu** [reply]
Hi @ebndev , yes, I understand what you mean. I hope this bug wont bother us again since this can literally be primary source of income for people

**Reply by rotimi-best** [reply]
I am also facing a pending payout issue.

My payout usually got into my bank account on the 20th but I haven't received it till today.

**Reply by theonlymo** [reply]
Same for me. My payouts usually get into my bank on the 22th, but nothing is triggered this month. (ticket  #2896252)
I also noticed that the email wording from sponsorships is different. in the past it used to look like this 
![image](https://github.com/user-attachments/assets/b9771c59-36ed-48f7-b5a5-6a4b7ba56216)

now it's like this 
![image](https://github.com/user-attachments/assets/85d11a77-370b-43a2-9327-3cfbb905b02a)

Do you have any idea what changed?

**Comment by amitguptagwl** [comment]
I'm facing this issue since february 2026 till date.

**Reply by slincoln75** [reply]
@amitguptagwl I'm not seeing a balance on your Sponsors account. Will you [open a ticket](https://support.github.com/) with our support team so we can look into it a bit more?

**Reply by cuilei5205189** [reply]
Hi @slincoln75,

I’m experiencing a very similar issue.

My Sponsors balance is available in Stripe Express, but no payout has been initiated and the estimated payout date rolled over to the next month.

I opened support ticket #4513507 almost three weeks ago, but I still haven’t received any response.

Could you please check whether my ticket has reached the GitHub Sponsors team or whether my account is stuck on manual payout/manual review?

Thank you.

**Reply by amitguptagwl** [reply]
> @amitguptagwl I'm not seeing a balance on your Sponsors account. Will you [open a ticket](https://support.github.com/) with our support team so we can look into it a bit more?

Thanks for your response. Please check for [NaturalIntelligence](https://github.com/NaturalIntelligence/). I've raised ticket too.

**Reply by amitguptagwl** [reply]
I'm not being paid since Feb 2026. I checked on strip, it says following.

> Update your bank account
> Your bank account has an error. Update your payout information to resume payouts to your bank account.

But bank detail is fine there. I've clicked on confirm button 5 times in last 5 months. but nothing happening. When I click on support link, they ask me to drop an email to `support@github.com` or to raise ticket on Github. If I drop email, it ask to raise ticket on Github. I raised ticket 3 weeks ago. but no single response yet.

#4578063

**Reply by slincoln75** [reply]
👋🏼 @amitguptagwl - I'm seeing that there's an error with your bank account according to Stripe which has resulted in your payouts being paused. Let me work with the support team so they can get in contact with you through your ticket for next steps.

**Reply by amitguptagwl** [reply]
By manual payout initiated by your team, it is solved. This took more than 3 weeks to get single response on the ticket until I highlighted here. I would really appreciate if response system can be improved.
